docs/source/tutorial_ph.md

```diff
@@ -0,0 +1,258 @@
+
+It is known that if the LTI system is minimal and stable, the following are equivalent:
+
+- The system is passive.
+- The system is port-Hamiltonian.
+- The system is positive real.
+
+See for example {cite}`BU22` for more details.
+
+In pyMOR, there exists a {{ PHLTIModel }} class. Currently, pyMOR only supports
+port-Hamiltonian systems with nonsingular E. {{ PHLTIModel }} inherits from
+{{ LTIModel }}, so {{ PHLTIModel }} can be used with all reductors that expect
+an {{ LTIModel }}. For model reduction, it is often desirable to preserve the
+port-Hamiltonian structure, i.e., to compute a ROM that is also port-Hamiltonian.
+
+If desired, a passive {{ LTIModel }} can be converted into a {{ PHLTIModel }} using
+the {meth}`~pymor.models.iosys.PHLTIModel.from_passive_LTIModel` method.
+Consequentely, one option to preserve port-Hamiltonian structure is to use a reductor
+that preserves passivity (but returns a ROM of type {{ LTIModel }}) and convert the
+ROM into a {{ PHLTIModel }} in a post-processing step.
+
+## A toy problem: Mass-spring-damper chain
+
+As a toy problem, we use a mass-spring-damper chain, which can be formulated
+as a port-Hamiltonian system (see {cite}`GPBV12`):
+
+```{image} msd_example.svg
+:alt: MSD example
+:width: 60%
+```
+
+Here, the spring constants are denoted by {math}`k_i` and the damping constants by
+{math}`c_i`, {math}`i=1,\dots,n/2`.
+The inputs {math}`u_1` and {math}`u_2` are the external forces on the first two
+masses {math}`m_1` and {math}`m_2`. The system outputs {math}`y_1` and {math}`y_2`
+correspond to the velocities of the first two masses {math}`m_1` and {math}`m_2`.
+The toy problem is included in pyMOR in the {mod}`pymor.models.examples` module as
+{func}`~pymor.models.examples.msd_example`.
+
+## Structure-preserving model order reduction
+
+pyMOR provides three reductors which can be used for model order reduction
+while preserving the port-Hamiltonian structure:
+
+- pH-IRKA ({class}`~pymor.reductors.ph.ph_irka.PHIRKAReductor`) {cite}`GPBV12`,
+- PRBT ({class}`~pymor.reductors.bt.PRBTReductor`) {cite}`DP84,GA04,HJS94`,
+- passivity preserving model reduction via spectral factorization
+  ({class}`~pymor.reductors.spectral_factor.SpectralFactorReductor`) {cite}`BU22`.
+
+In this section, we apply all three reductors on our toy example and compare
+their performance. All three reductors are described in {cite}`BU22` in more detail.
+
+Note: Currently, the {class}`~pymor.reductors.bt.PRBTReductor` and
+{class}`~pymor.reductors.spectral_factor.SpectralFactorReductor` reductors require
+the symmetric part of {math}`D` (i.e., the $S$ matrix in the port-Hamiltonian system)
+to be nonsingular. The MSD example has a zero {math}`D` matrix. Therefore,
+we have to add a small regularization feedthrough term, i.e., we replace $D$ with
+$D+\varepsilon I_m$. This is a limitation of the current implementation since the
+numerical solution of the KYP-LMI is obtained by solving a related Riccati
```

Comment:
Maybe simply write the Riccati equation A^T X E + E^T X A + (C^T − X B)(D + D^T)^{-1}(C − B^T X) = 0 here. Then it is more clear what this small regularization term is for and why it is necessary.